### PR TITLE
STM32U5: Add PWR peripheral

### PR DESCRIPTION
**Pull request by RootCubed**
Not quite ready yet - there are 4 registers that aren't in any SVD files, so I'll probably have to manually check on which devices they are available...

(FORCE_USBPWR, JPEGRAMPDS, VDD11USBDIS and SRAM6PDSx, see last lines of pwr_u5.yaml)

### CODE REVIEW

**Comment by RootCubed** [comment]
Well that was painful... I was able to use [AN5766](https://www.st.com/resource/en/application_note/an5766-migrating-within-stm32u5-series-microcontrollers-stmicroelectronics.pdf) as a guide but it was still quite annoying to go though those SVDs to check where it was correct and where not. Hopefully I got it right now.

File: devices/stm32u575.yaml [hunk 1]
```diff
@@ -9,6 +9,60 @@ _modify:
   ADC12:
     name: ADC12_Common
 
+PWR:
+  CR2:
+    _remove: PKARAMPDS
+  _add:
+    CR4:
+      displayName: CR4
```
Comment (RootCubed) [inline devices/stm32u575.yaml:17]:
```suggestion
      displayName: PWR control register 4
```
Will apply all suggestions at the end.

File: devices/stm32u585.yaml [hunk 2]
```diff
@@ -9,6 +9,58 @@ _modify:
   ADC12:
     name: ADC12_Common
 
+PWR:
+  _add:
+    CR4:
+      displayName: CR4
```
Comment (RootCubed) [inline devices/stm32u585.yaml:15]:
```suggestion
      displayName: PWR control register 4
```
Will apply all suggestions at the end.

**Comment by burrbull** [comment]
Rebase on master branch, please.

**Comment by RootCubed** [comment]
Would you like me to squash the commits so far or leave them separate for now?

**Comment by burrbull** [comment]
Let's squash.

**Review by burrbull — APPROVED** [review]

**Pull request merged**